cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/ZooKeeperReconciler.java

```diff
@@ -471,14 +467,8 @@ protected Future<Void> podDisruptionBudget() {
      *
      * @return  Completes when the PDB was successfully created or updated
      */
-    protected Future<Void> podDisruptionBudgetV1Beta1() {
-        if (pfa.hasPodDisruptionBudgetV1()) {
-            return Future.succeededFuture();
-        } else {
-            return podDisruptionBudgetV1Beta1Operator
-                    .reconcile(reconciliation, reconciliation.namespace(), KafkaResources.zookeeperStatefulSetName(reconciliation.name()), zk.generatePodDisruptionBudgetV1Beta1())
-                    .map((Void) null);
-        }
+    protected Future<Void> podDisruptionBudgetV1() {
```

Comment:
This seems wrong => this does not do anything and should be removed. The v1 version of the PDB is managed by the method right above.